src/redux/actionCreators.js

```diff
@@ -18,9 +18,24 @@ const updateNewMessageBodyCreator = (payload) => {
 
 const updateSendMessageCreator = () => ({ type: ACTION_TYPES.SEND_MESSAGE });
 
+const updateFollowStatusAC = (userId) => ({
+  type: ACTION_TYPES.FOLLOW_TO_USER,
+  userId,
```

Comment:
Consider keeping an action body generic. Every action should have 2 fields: type and payload.
So add "payload" key and move "userId" under it.
`{ type: '...', payload: userId, }`

Reply:
done